### ISSUE TITLE
T10.25. Чи може клас мати кілька деструкторів?

### ISSUE THREAD

**Issue by WAG13** [kzhereb/knu-is-ooop2019#25]

**Comment by JuliaSamartseva** [comment]
Незважаючи на те, що клас може мати декілька конструкторів, деструктор може бути лише один.

**Comment by i1red** [comment]
Ні

**Comment by solovdaria** [comment]
Ні

**Comment by Velly69** [comment]
Ні, тому що деструктори не можуть приймати аргументів. З цього випливає, що для кожного класу існує лише один деструктор, тому що не існує можливості перегрузити деструктори як функції та відрізнятися один від одного аргументами вони не можуть.

**Comment by AntonHladkiy** [comment]
Ні

**Comment by RostyslavMV** [comment]
Не може

**Comment by ilyakorenevskyi** [comment]
Ні, не може. Деструктор не можна перевантажити (не приймає аргументи)

**Comment by WAG13** [comment]
Ні, не може

**Comment by Vladyslav-Samokryk** [comment]
Не може, має бути тільки один деструктор.
